packages/vuetify/src/components/VDatePicker/VDatePickerHeader.ts

```diff
@@ -112,6 +112,9 @@ export default mixins(
       const header = this.$createElement('div', this.setTextColor(color, {
         key: String(this.value)
       }), [this.$createElement('button', {
+        domProps: {
```

Comment:
attrs

Reply:
I think he means you should use attrs instead of domProps.

Reply:
Oh, my bad. Of course I changed it.